src/main/java/frc/robot/vision/multivisionsources/MultiAprilTagVisionSources.java

```diff
@@ -53,24 +54,24 @@ public MultiAprilTagVisionSources(
 		this(logPath, robotHeadingSupplier, useRobotHeadingForPoseEstimating, List.of(visionSources));
 	}
 
-	private void updateAngleInHeadingRequiringSources(GyroAngleValues gyroAngleValues) {
+	private void updateAngleInHeadingRequiringSources(RobotAngleValues robotAngleValues) {
 		for (VisionSource<AprilTagVisionData> visionSource : visionSources) {
 			if (visionSource instanceof RobotHeadingRequiringVisionSource robotHeadingRequiringVisionSource) {
-				robotHeadingRequiringVisionSource.updateGyroAngleValues(gyroAngleValues);
+				robotHeadingRequiringVisionSource.updateRobotAngleValues(robotAngleValues);
 			}
 		}
 	}
 
 	private void updateAngleInHeadingRequiringSources(Rotation3d angle, double yawRate, double pitchRate, double rollRate) {
-		updateAngleInHeadingRequiringSources(new GyroAngleValues(angle, yawRate, pitchRate, rollRate));
+		updateAngleInHeadingRequiringSources(new RobotAngleValues(angle, yawRate, pitchRate, rollRate));
```

Comment:
```suggestion
		updateAngleInHeadingRequiringSources(new RobotAngles(angle, yawRate, pitchRate, rollRate));
```

or RobotHeadings i like short names. your choice

Reply:
Well it's not just heading/angles, but also the rate which each of them is changing.